types/office-js/index.d.ts

```diff
@@ -24852,13 +24936,28 @@ declare namespace Excel {
          */
         layout: "Table";
     }
+    /**
+     * Represents a section of a card that is arranged as two columns in card view.
+     *
+     * @remarks
+     * [Api set: ExcelApi 1.19]
+     */
+    interface CardLayoutTwoColumnSection extends CardLayoutSectionStandardProperties {
+        /**
+         * Represents the type of layout for this section.
+         *
+         * @remarks
+         * [Api set: ExcelApi 1.19]
+         */
+        layout: "TwoColumn";
+    }
     /**
      * Represents the layout of a section of a card in card view.
      *
      * @remarks
      * [Api set: ExcelApi 1.16]
      */
-    type CardLayoutSection = CardLayoutListSection | CardLayoutTableSection;
+    type CardLayoutSection = CardLayoutListSection | CardLayoutTableSection | CardLayoutTwoColumnSection;
```

Comment:
same here - is this already in office-js-preview? If not, should it be?

Reply:
Yes, it's in office-js-preview